bin/keycloak-httpd-client-install

```diff
@@ -829,24 +833,29 @@ def build_template_params(options):
 
 def build_mellon_httpd_config_file(options):
```

Comment:
A function whose name includes "mellon" should not be producing data applicable to both mellon and oidc.